tst/EnergyPlus/unit/WindowManager.unit.cc

```diff
@@ -2835,6 +2835,8 @@ TEST_F(EnergyPlusFixture, WindowManager_SrdLWRTest)
 
     HeatBalanceSurfaceManager::InitSurfacePropertyViewFactors(*state);
 
+    HeatBalanceSurfaceManager::GetSurroundingSurfacesTemperatureAverage(*state);
+
```

Comment:
New function for calculating the view factor weighted average temperature of the surrounding surfaces. The average surface temperature is computed every timestep for all applicable surfaces.